client/ayon_traypublisher/plugins/create/create_csv_ingest.py

```diff
@@ -644,9 +644,21 @@ def _add_representation(
         # convert ### string in file name to %03d
         # this is for correct frame range validation
         # example: file.###.exr -> file.%03d.exr
+        file_head = basename.split(".")[0]
         if "#" in basename:
             padding = len(basename.split("#")) - 1
-            basename = basename.replace("#" * padding, f"%0{padding}d")
+            seq_padding = f"%0{padding}d"
+            basename = basename.replace("#" * padding, seq_padding)
+            file_head = basename.split(seq_padding)[0]
+            is_sequence = True
+        if "%" in basename:
```

Comment:
Should this be `elif`? Because after the first if the `%` will be in the basename and will then ALSO trigger that logic. Not sure if intentional?

Reply:
Agree this will run if the filename had `#` in filename, which is not needed, or?

Reply:
done